src/Nancy/ModelBinding/BindingConfig.cs

```diff
@@ -43,5 +43,11 @@ public BindingConfig()
         /// </summary>
         /// <value><see langword="true" /> if the binder is allowed to overwrite non-default values, otherwise <see langword="false" />.</value>
         public bool Overwrite { get; set; }
+
+        /// <summary>
+        /// If the string passed in is empty the binder will not bind it.
+        /// </summary>
+        /// <value><see langword="true" /> if the binder will convert empty strings to null, otherwise <see langword="false" />.</value>
+        public bool ConvertEmptyStringToNull { get; set; }

```

Comment:
ConvertEmptyStringToNull was what was used in ASP.NET MVC, it's what I found when I was trying to find a solution to this issue. Not that it is any better but that's why I named it that.